Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "amd-apcb"
 version = "0.3.0"
 authors = ["Oxide Computer"]
-edition = "2018"
+edition = "2021"
```

Comment:
This causes an extra warning that TryFrom now is auto-imported.

Reply:
Fixed.